### PR TITLE
Set User#zooniverse_id for panoptes-only users

### PR DESCRIPTION
**Pull request by parrish**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -53,6 +53,7 @@ class User < ActiveRecord::Base
   before_validation :setup_unsubscribe_token, on: [:create]
   before_validation :update_ouroboros_created
   before_save :update_ouroboros_created
+  after_create :set_zooniverse_id
```
Comment (camallen) [inline app/models/user.rb]: I assume we don't want to add this to migrated users.

Comment (edpaget) [inline app/models/user.rb]: It doesn't look like it will since it happens with the ||=

Comment (parrish) [inline app/models/user.rb]: Yep.  Just panoptes users that don't have a zooniverse_id

**Comment by camallen** [comment]
LGTM

**Pull request merged**